packages/ui-components/styles/markdown.css

```diff
@@ -90,10 +90,6 @@ main {
     &:has(code) {
       @apply max-xs:decoration-neutral-800
         dark:max-xs:decoration-neutral-200;
-
```

Comment:
IMO this should be made within the relevant component instead of `markdown.css`. For example, code links within the markdown content have also been unintentionally affected by this change;

Before:
<img width="884" alt="image" src="https://github.com/user-attachments/assets/1d5c2701-3e7b-44ec-8579-95914ae27eb4" />

Preview:
<img width="884" alt="image" src="https://github.com/user-attachments/assets/462f4b37-7a6b-40c5-a82d-03fcc2dc7192" />

Source;
`/en/learn/getting-started/introduction-to-nodejs#an-example-nodejs-application`

Reply:
I see, i'll update `AlertBox` (the affected component)

Reply:
Looking at the css again, it seems it's supposed to _only_ apply that to the `a` elements, so maybe there's a selector issue, i'll investigate